docs/src/api/class-response.md

```diff
@@ -43,6 +43,26 @@ Headers with multiple entries, such as `Set-Cookie`, appear in the array multipl
 An array with all the request HTTP headers associated with this response. Unlike [`method: Response.allHeaders`], header names are not lower-cased.
 Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
 
+## async method: Response.getHeaderValue
+- returns: <[null]|[string]>
+
+Returns the first value of the header matching the name. The name is case insensitive. If multiple headers have the same name, for example `set-cookie`, the first is returned. If no header is found, `null` is returned.
```

Comment:
We should concatenate values using `', '` as a separator instead.

Reply:
I thought the reason why set cookie can be specified more than once is because cookies can have ',' in their values. How would someone use this?

Reply:
+1 to Joel but [`Headers.get`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/get) is also doing this weird comma separation logic :/

Reply:
By the spec, accept-encoding: gzip, deflate is indistinguishable from accept-encoding: gzip\r\naccept-encoding: deflate. Yet you would want to get "gzip, deflate" as the value.

Reply:
done